week5/src/pages/ContentPage.tsx

```diff
@@ -0,0 +1,74 @@
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import styled from 'styled-components';
+import { StyledInput } from '../components/Input';
+import Layout from '../components/Layout';
+import { useGetLetterHook } from '../hooks/useGetLetterHook';
+
+const ContentPage = () => {
+  const [input, setInput] = useState('');
+  const [IntLetterId, setIntLetterId] = useState<number>(0);
+  const [check, setCheck] = useState<boolean>(false);
+  const { letterId } = useParams();
+  const { letters } = useGetLetterHook();
+
+  useEffect(() => {
+    if (letterId === undefined) return;
+    setIntLetterId(parseInt(letterId));
+  }, [letterId]);
+
+  const onChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setInput(e.target.value);
+  };
+
+  const onSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    if (letterId === undefined) return;
+    if (input === letters?.[IntLetterId].password) {
+      setCheck(true);
+    } else {
+      setCheck(false);
+    }
```

Comment:
```suggestion
if (letterId ) 
        if (input === letters[IntLetterId].password) {
      setCheck(true);
    } else {
      setCheck(false);
    }
```
로 바꾸면 옵셔널 체이닝 ? 안써도데용!